src/Contract/Operation/IsNotEmptyable.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace loophp\collection\Contract\Operation;
+
+/**
+ * @template TKey
+ * @template T
+ */
+interface IsNotEmptyable
+{
+    /**
+     * Check if a collection has any elements inside.
```

Comment:
please make this doc block description the exact same as the one in the docs [here](https://github.com/loophp/collection/pull/292/files?diff=split&w=1#diff-89f6324da13155a697ae0b42136e03a40892dd4edc4257ea1e451f0ea2cc3f80R1283)